### PR TITLE
Add GL_QCOM_texture_foveated_subsampled_layout

### PR DESCRIPTION
**Pull request by tate-hornbeck**

### CODE REVIEW

**Comment by tate-hornbeck** [comment]
@jackohound 

After building I'm seeing fetch_rate added to the headers

+#ifndef GL_QCOM_shader_framebuffer_fetch_rate
+#define GL_QCOM_shader_framebuffer_fetch_rate 1
+#endif /* GL_QCOM_shader_framebuffer_fetch_rate */

**Comment by jackohound** [comment]
@tate-hornbeck, yes I believe this is expected.  As per @oddhack 's request in [PR#207](https://github.com/KhronosGroup/OpenGL-Registry/pull/207), I created that PR without the header updates.  PR#207 is now merged, but the associated header updates haven't been committed.

Maybe @oddhack can comment how he wants to handle this.   Is the expectation that all PRs should be created without the header updates?

**Comment by oddhack** [comment]
Including generated files in PRs invariably leads to conflicts with multiple PRs in flight, so I prefer to do it myself after accepting outstanding PRs - when you've removed them from this PR I'll accept it & update the headers.

File: extensions/QCOM/QCOM_texture_foveated_subsampled_layout.txt [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+
+    Get Value                              Type    Get Command  Minimum Value   Description               Sec
+    ---------                              ----    -----------  -------------   -----------               ------
+    MAX_SHADER_SUBSAMPLED_IMAGE_UNITS_QCOM  Z+      GetIntegerv  1               No. of subsampled texture 8.19
+                                                                                 images allowed in any
+                                                                                 shader stage.
+
+Errors
+
+    INVALID_ENUM is generated by TexParameter{if} or TexParamter{if}v
```
Comment (janharaldfredriksen-arm) [inline extensions/QCOM/QCOM_texture_foveated_subsampled_layout.txt]: typo: TexParamter (repeated a few times below)

Comment (tate-hornbeck) [reply extensions/QCOM/QCOM_texture_foveated_subsampled_layout.txt]: Thanks

**Comment by pdaniell-nv** [comment]
Approved to merge by the OpenGL/ES working group. @oddhack please merge if it looks good to you.

**Comment by oddhack** [comment]
Merging - I'll have to correct the ES extension # though as that was pre-empted by another vendor extension.

**Pull request merged**